usrtools/txgen/app/cli-functions.c

```diff
@@ -276,14 +347,21 @@ pcap_cmd(int argc, char **argv)
     return 0;
 }
 
-static struct cli_map start_map[] = {{10, "start %P"}, {20, "stop %P"}, {-1, NULL}};
+static struct cli_map start_map[] = {{10, "start %P"},
+                                     {20, "stop %P"},
+                                     {50, "start %P latsampler"},
+                                     {60, "stop %P latsampler"},
+                                     {-1, NULL}};
 
 static const char *start_help[] = {
     "",
     "start <portlist>                   - Start transmitting packets",
     "stop <portlist>                    - Stop transmitting packets",
     "stp                                - Stop all lports from transmitting",
     "str                                - Start all lports transmitting",
+    "start <portlist> latsampler        - Start latency sampler, make sure to set sampling "
+    "parameters before starting",
```

Comment:
Does this line need to adjusted to be under the Start word?

Reply:
it looks like clang format doesn't like this:

```
--- ./usrtools/txgen/app/cli-functions.c	(original)
+++ ./usrtools/txgen/app/cli-functions.c	(reformatted)
@@ -360,7 +360,7 @@
     "stp                                      - Stop all lports from transmitting",
     "str                                      - Start all lports transmitting",
     "start <portlist> latsampler|lat|latency  - Start latency sampler, make sure to set sampling "
-                                                "parameters before starting",
+    "parameters before starting",
     "stop <portlist> latsampler|lat|latency   - Stop latency sampler, dumps to file if specified",
     CLI_HELP_PAUSE,
     NULL};
```